packages/inter-protocol/src/proposals/demoIssuers.js

```diff
@@ -288,6 +288,11 @@ export const connectFaucet = async ({
             case Stake.symbol:
               return bldIssuerKit;
             default: {
+              // This nested await is safe because "terminal-control-flow".
```

Comment:
Not production code in any case.

Reply:
Good to know. That's what I suspected from the name. But OTOH it is in a src/ directory, not test/ or demo/.
OTTH, as example code it needs to set the right precedents for how people will write production code.